src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -33,6 +37,7 @@ public abstract class PythonNodeBase : VariableInputNode
         protected PythonNodeBase(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
         {
             ArgumentLacing = LacingStrategy.Disabled;
+            pythonEngineVersionsList = Enum.GetValues(typeof(PythonEngineVersion)).Cast<PythonEngineVersion>();
```

Comment:
Please notice we are re-initializing a static field each time a Python node is created.

Also, adding a field to our code just to be used in the tests is not something we should do if we can avoid it. In this case it seems easy to avoid, we just need to a helper method in our test class.

Reply:
Initially, I was using a static list in the tests to get the values. But the tests were failing because it couldn't find the PythonNodeModels.dll (the enum is defined in here) at runtime. May be the static field was the problem. Getting the list of values inside a function worked. So changed it back. Thanks @mmisol.